docs/SymbolInstance.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Symbol Instance
+order: 210
+section: components
+---
+
+```javascript
+var SymbolInstance = sketch.SymbolInstance
+```
+
+```javascript
+import { SymbolInstance } from 'sketch-api'
+```
+
+A [Symbol](https://www.sketchapp.com/docs/symbols/) instance. It is an instance of [Layer](#layer) so all the methods defined there are available.
```

Comment:
As with the other PRs, the use of "it is only returns when" reads a little odd. Would either "it is only returned when" or "it only returns when" be correct?